### DISCUSSION TITLE
scenario/%E7%B3%BB%E7%BB%9F%E8%AE%BE%E8%AE%A1%E9%9D%A2%E8%AF%95%E6%8C%87%E5%8D%97/01-short-url-design

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#964] [Announcements]
# scenario/%E7%B3%BB%E7%BB%9F%E8%AE%BE%E8%AE%A1%E9%9D%A2%E8%AF%95%E6%8C%87%E5%8D%97/01-short-url-design

短链URL生成器是一个互联网媒体分享最常用的方式了，在Java面试中关于短链设计的考题是高频考点，接下来就描述一下短链设计的具体思路。

https://topgeeky-site.vercel.app/scenario/%E7%B3%BB%E7%BB%9F%E8%AE%BE%E8%AE%A1%E9%9D%A2%E8%AF%95%E6%8C%87%E5%8D%97/01-short-url-design.html

**Comment by 1213RockLee** [comment]
大佬写的真好，能再多给些内容么，最近要面试，准备下场景题目
